src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
```

Comment:
If you start this, then you'll have to do this for all of em (Filt, Attr, Prop, Oper), (and I'd have done the updates as well, but since you seem to have started..)